docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -763,6 +763,42 @@ Default value: 10000.
 <max_thread_pool_size>12000</max_thread_pool_size>
 ```
 
+## max_thread_pool_free_size {#max-thread-pool-free-size}
+
+The number of threads that are free in the Global Thread pool.
+
+Default value: 1000.
+
+**Example**
+
+``` xml
+<max_thread_pool_free_size>1200</max_thread_pool_free_size>
+```
+
+## thread_pool_queue_size {#thread-pool-queue-size}
+
+The limit to the number of jobs that can be scheduled on the Global Thread pool.
+
+Default value: 10000.
```

Comment:
It should be noticed that increasing this number will lead to more memory consumption. Better to set it to the value equals to max_thread_pool_size if the latter is changed.